modules/trusted-firmware-m/tfm_boards/common/nrf_provisioning.c

```diff
@@ -16,6 +16,51 @@
 #include "nrf_provisioning.h"
 #include <identity_key.h>
 #include <tfm_spm_log.h>
+#ifdef NRF53_SERIES
+#include <dfu/pcd_common.h>
+#include <spu.h>
+#include <hal/nrf_reset.h>
+
+static struct pcd_cmd *cmd = (struct pcd_cmd *)PCD_CMD_ADDRESS;
```

Comment:
```suggestion
volatile static struct pcd_cmd *cmd = (struct pcd_cmd *)PCD_CMD_ADDRESS;
```

If the content might be modified by another core, I think it should be volatile.